nofish/NF_ReaScript.cpp

```diff
@@ -511,3 +511,13 @@ void NF_Base64_Encode(const char* str, int str_sz, const bool usePadding, char*
 	const char* encoded = b64.Encode(str, str_sz, usePadding);
 	CopyToBuffer(encoded, encodedStrOut, encodedStrOut_sz);
 }
+
+const char* JB_GetSWSExtraProjectNotes(ReaProject* project)
```

Comment:
These wrappers in this file are not needed, ReaScript can directly use the ones in SnM_Notes.cpp.